packages/widget/src/state/wallets.ts

```diff
@@ -66,6 +79,43 @@ export const knownEthermintLikeChains = [
   "shido_9008-1",
 ];
 
+const DEEPLINK_CHOICE = "WALLETCONNECT_DEEPLINK_CHOICE";
+const RECENT_WALLET_DATA = "WCM_RECENT_WALLET_DATA";
+
+export const walletConnectDeepLinkByChainTypeAtom = atomWithStorageNoCrossTabSync(
+  "WC_DEEPLINK_BY_CHAIN_TYPE",
+  {
+    [ChainType.Cosmos]: {
+      deeplink: "",
+      recentWalletData: "",
+    },
```

Comment:
even if cosmos is unnecessary, it makes the code/typing more straight forward to also have a key for cosmos